Bluepill-cli/Bluepill-cli/Simulator/SimulatorHelper.m

```diff
@@ -63,18 +63,32 @@ + (NSDictionary *)appLaunchEnvironmentWithBundleID:(NSString *)hostBundleID
                                             config:(BPConfiguration *)config {
     NSString *hostAppExecPath = [SimulatorHelper executablePathforPath:config.appBundlePath];
     NSString *testSimulatorFrameworkPath = [[hostAppExecPath stringByDeletingLastPathComponent] stringByDeletingLastPathComponent];
-    NSString *dyldLibraryPath = [NSString stringWithFormat:@"%@/Platforms/iPhoneSimulator.platform/Developer/Library/Frameworks", config.xcodePath];
     NSMutableDictionary<NSString *, NSString *> *environment = [@{
-//                                                                  @"DYLD_PRINT_ENV": @YES,
-//                                                                  @"DYLD_PRINT_LIBRARIES": @YES,
                                                                   @"DYLD_FALLBACK_FRAMEWORK_PATH" : [NSString stringWithFormat:@"%@/Library/Frameworks:%@/Platforms/iPhoneSimulator.platform/Developer/Library/Frameworks", config.xcodePath, config.xcodePath],
-                                                                  @"DYLD_FRAMEWORK_PATH" : dyldLibraryPath,
+                                                                  @"DYLD_FALLBACK_LIBRARY_PATH" : [NSString stringWithFormat:@"%@/Platforms/iPhoneOS.platform/Developer/Library/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/usr/lib", config.xcodePath],
                                                                   @"DYLD_INSERT_LIBRARIES" : [NSString stringWithFormat:@"%@/Platforms/iPhoneOS.platform/Developer/Library/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/Developer/usr/lib/libXCTTargetBootstrapInject.dylib", config.xcodePath],
-                                                                  @"DYLD_LIBRARY_PATH" : dyldLibraryPath,
+                                                                  @"DYLD_LIBRARY_PATH" : [NSString stringWithFormat:@"%@/Platforms/iPhoneSimulator.platform/Developer/Library/Frameworks", config.xcodePath],
+                                                                  @"DYLD_ROOT_PATH" : [NSString stringWithFormat:@"%@/Platforms/iPhoneOS.platform/Developer/Library/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot", config.xcodePath],
                                                                   @"NSUnbufferedIO" : @YES,
                                                                   @"OS_ACTIVITY_DT_MODE" : @YES,
-                                                                  @"MNTF_TINKER_DELAY": @0.01,
+                                                                  @"NSUnbufferedIO" : @YES,
+                                                                  @"MNTF_SCREENSHOTS" : @"$(PROJECT_DIR)/build/outputs/tests_artifacts",
```

Comment:
I'm not sure we should be hardcoding all of these MNTF values. They can be passed in the environment section of the config file.

Reply:
yeah, they should not be set here.